addon/components/ember-notify/message.js

```diff
@@ -2,6 +2,10 @@ import Ember from 'ember';
 import layout from '../../templates/components/ember-notify/message';
 import Notify from 'ember-notify';
 
+function checkSendCloseIntent() {
+  if (this.get('isDestroyed')) return;
```

Comment:
I had tried that. At the time `closeIntent` fires the component is not yet destroyed. `isDestroyed` is true 100 ms later: https://github.com/aexmachina/ember-notify/pull/72/files#diff-d5a2e62748aaf28e7439e392422147deR62
I can make this function a method of the component instance if that would be more in keeping with the project. The main reason I broke it into a function is that https://github.com/aexmachina/ember-notify/pull/72/files#diff-d5a2e62748aaf28e7439e392422147deL38 had identical functionality.